packages/@react-aria/table/docs/useTable.mdx

```diff
@@ -800,6 +804,645 @@ let rows = [
+    column,
+    label: 'Resizer',
+    onResizeStart,
+    onResize,
+    onResizeEnd,
+    triggerRef
+  }, layoutState, ref);
+
+  return (
+    <div
+      role="presentation"
+      style={{
+        cursor: undefined,
+        width: '6px',
+        height: 'auto',
+        border: '2px',
+        borderStyle: 'none solid',
+        borderColor: layoutState.resizingColumn === column.key ? 'orange' : 'grey',
+        touchAction: 'none',
+        flex: '0 0 auto',
+        boxSizing: 'border-box',
+        visibility: showResizer ? 'visible' : 'hidden',
+        marginLeft: '4px'
+      }}
+      {...resizerProps}>
+      <VisuallyHidden>
+        <input
+          ref={ref}
+          type="range"
+          {...inputProps} />
+      </VisuallyHidden>
+    </div>
+  );
+});
+```
+
+### MenuTrigger
+
+The `MenuTrigger` is used in the `ResizableTableColumnHeader` to display a list of available actions available to a column. It is built using the [useMenuTrigger](useMenu.html)
+and [useMenuTriggerState](/react-stately/useMenuTriggerState.html) hooks, and can be shared with many other components. Note that you'll have to call `continuePropagation()` in `onKeyDown`
+to properly bubble the `keydown` event up to `useSelectableCollection` for grid keyboard navigation.
+
+<details>
+  <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show code</summary>
+
+```tsx example export=true render=false
+import {useMenuTriggerState} from '@react-stately/menu';
+import {useMenuTrigger} from '@react-aria/menu';
+import {Item} from '@react-stately/collections';
+
+// Reuse the Button, Menu, and Popover from your component library. See below for details.
+import {Button, Menu, Popover} from 'your-component-library';
+
+const MenuTrigger = React.forwardRef((props, ref) => {
+  // Create state based on the incoming props
+  let state = useMenuTriggerState(props);
+
+  // Get props for the button and menu elements
+  let {menuTriggerProps, menuProps} = useMenuTrigger({trigger: 'press'}, state, ref);
+  /*- begin highlight -*/
+  // Continue propagation of keydown event so that the left/right arrow key presses properly bubble to the table (useSelectableCollection)
+  let onKeyDown = (e) => e.continuePropagation();
+  /*- end highlight -*/
+  return (
+    <>
+      <Button
+        {...mergeProps(menuTriggerProps, {onKeyDown})}
+        buttonRef={ref}
+        style={{height: 30, fontSize: 14, ...props.style}}>
+        {props.label}
+      </Button>
+      {state.isOpen &&
+        <Popover state={state} triggerRef={ref} placement="bottom start">
+          <Menu
+            {...props}
+            {...menuProps} />
+        </Popover>
+      }
+    </>
+  );
+});
+```
+
+</details>
+
+### Menu
+
+The `Menu` is used to display the various actions available to the column, rendered with a `Popover` when the user presses the column header.
+This is built using the [useMenu](useMenu.html) and [useTreeState](/react-stately/useTreeState.html) hooks.
```

Comment:
Stately links don't work in the build. I think they will still work in prod, but maybe safer to use relative path like `[useTreeState](../react-stately/useTreeState.html)`?